### ISSUE TITLE
JS time different in UI and text value when TimeZone is not as local machine

### ISSUE THREAD

**Issue by jonmifsud** [hananils/datetime#172]
I'm working with a US client so I've set up everything with their timezone, everything works (for me) other than the Date UI hours. Midnight shows as 6am (it adds the time-difference showing my local time) I'm not sure if this is the correct way of showing it, as the date string would read 12:00 am but the slider 06:00 am. It's not a big issue for me but might be for anyone else encountering it.

**Comment by nitriques** [comment]
Well, well, well... We did tried to fix those issues but could not.
Are you seeing the problem ? Or the client is ?

**Comment by jonmifsud** [comment]
Client is good, site is set up in NewYork time and they're based in New York, I'm seeing the problem as I'm based in Malta, so I could probably debug it if you have any pointers it would be appreciated.

**Comment by nitriques** [comment]
Yeah, that's a bug, i.e. the server and client is not in the right time zone.

Support for this in plain javascript sucks ass. We should use moment (as the core does) to fix this.

Or have fun with dates: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC
